motor/docstrings.py

```diff
@@ -1268,3 +1268,32 @@ async def coro():
 .. _$expr: https://docs.mongodb.com/manual/reference/operator/query/expr/
 .. _$where: https://docs.mongodb.com/manual/reference/operator/query/where/
 """
+
+create_data_key_doc = """Create and insert a new data key into the key vault collection.
+
+Takes the same constructors as 
+:class:`pymongo.encryption.ClientEncryption.create_data_key`,
+with only the following slight difference using async syntax:
+
+:Parameters:
+  - `key_alt_names` (optional): An optional list of string alternate
+    names used to reference a key. If a key is created with alternate
+    names, then encryption may refer to the key by the unique alternate
+    name instead of by ``key_id``. The following example shows creating
+    and referring to a data key by alternate name::
+
+      await client_encryption.create_data_key("local", keyAltNames=["name1"])
+      # reference the key with the alternate name
+      await client_encryption.encrypt("457-55-5462", keyAltName="name1",
+                                algorithm=Algorithm.Random)
```

Comment:
Algorithm.Random -> Algorithm.AEAD_AES_256_CBC_HMAC_SHA_512_Random

Algorithm.Random does not exist. Could you also open an issue to fix this mistake in the pymongo docs?